kedro-org/kedro
# Update Kedro banner image

## Description

Resolves https://github.com/kedro-org/kedro-viz/issues/1393.

## QA notes

Here's the README local to this branch: https://github.com/kedro-org/kedro/tree/chore/update-banner-images.

We've also got a light and dark theme banner working, per [this article](https://dev.to/mateusabelli/make-images-responsive-to-lightdark-mode-on-github-3gbg) Yetu found.

Can confirm it's working on the dark theme!

<img width="913" alt="image" src="https://github.com/kedro-org/kedro/assets/16869061/39243e53-aa5f-4138-ac69-e3f02e48dc9e">


## Checklist
